tox.ini

```diff
@@ -4,7 +4,7 @@ envlist =
 
 [testenv]
 commands =
-    python setup.py -q test -q
+    python setup.py -q test -q []
```

Comment:
It is. `[]` passes the positional arguments from the command line through. There's a longer way to spell it, something like `{{posargs}}` or something, but I can never remember it.